packages/nouns-api/src/graphql/utils/queryUtils.ts

```diff
@@ -63,3 +63,7 @@ export const DATE_FILTERS: { [key: string]: any } = {
     }),
   },
 };
+
+export const getIsArchived = (idea: any) => {
+  return moment(idea.createdAt).isBefore(moment().subtract(7, 'days').toISOString());
+};
```

Comment:
I was thinking we would include this in the virtual tag map.
Although I understand we haven't merged our code yet.

```
NEW: {
    type: 'ARCHIVED',
    label: 'Archived',
    color: 'yellow',
    filterFn: (idea: Idea) => {
      return moment(idea.createdAt).isBefore(moment().subtract(7, 'days').toISOString());
    },
  },
```

Reply:
Also, a bit confused by this code. Are we archiving ideas after 7 days? That doesn't seem very long. 
I feel like archiving should be a user event -- you declare your idea as archived

Reply:
> Also, a bit confused by this code. Are we archiving ideas after 7 days?

Yeah Ade wanted to do time based archiving, I got 7 days from somewhere but can't find it now. 14 days might be a better timeframe to start with. This can work alongside manual archiving but i think we can add support for that after we see how the time based approach is working.

Manual archiving can be a quick follow tbh shouldn't be a huge amount of work to add after and the time based approach will reduce the immediate need for it.

Reply:
I've updated the property name to "closed" as archived was causing confusion, this will also allow us to introduce a archived prop later for manual archiving if we want to actually hide ideas.